script.js

```diff
@@ -0,0 +1,62 @@
+function capitalize(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function reverseString(string) {
+  return string.split("").reverse().join("");
```

Comment:
Can you do think about a way to do this without using `split` and `reverse`

Reply:
I'll look into it